src/nativestore/NodeManager.cpp

```diff
@@ -480,7 +480,44 @@ std::map<long, std::unordered_set<long>> NodeManager::getAdjacencyList() {
         }
         adjacencyList.emplace((long)node->nodeId, neighbors);
     }
+    return adjacencyList;
+}
+
+std::map<long, std::unordered_set<long>> NodeManager::getCentralAdjacencyList() {
```

Comment:
Function body of `getCentralAdjacencyList` and `getLocalAdjacencyList(` looks same instead of the db name. I think its better to extract the common code to a common function

Reply:
done

Reply:
Ok